docs/versioning.md

```diff
@@ -0,0 +1,89 @@
+# Versioning
+
+## Introduction
+
+Fedora 4 provides support for making snapshots of the current state of a resource which in turn become part of the resources version history.  The RDF for historic version snapshots may be browsed and old non-RDF content may be downloaded.  Furthermore, an object or subgraph may be reverted to the state that it existed in a historic version.
```

Comment:
I would start by talking about Memento and not Fedora 4. There is no reason to give the details of how Fedora 4 works except as a means to distinguish it from the Versioning specification.